README.md

```diff
@@ -37,7 +54,215 @@ The easiest way to deploy your Next.js app is to use the [Vercel Platform](https
 
-## Base Stack
+## Main Dependencies
+
+VBS incorporates several popular libraries and tools:
+
+- [Next.js](https://nextjs.org/)
+- [Firebase](https://firebase.google.com/)
+- [HubSpot API Client](https://developers.hubspot.com/)
+- [Radix UI](https://radix-ui.com/)
+- [Tailwind CSS](https://tailwindcss.com/docs/utility-first)
+- [Headless UI](https://headlessui.dev/)
+- [Heroicons](https://heroicons.com/)
+- [Axios](https://axios-http.com/)
+- [clsx](https://www.npmjs.com/package/clsx)
+- [Lucide Icons for React](https://github.com/lucide-icons/lucide/tree/main/packages/lucide-react)
+- [React Toastify](https://fkhadra.github.io/react-toastify/)
+- And several others focused on development like ESLint configurations and PostCSS.
+
+## Usage
+
+### Local Development
+
+To run the VBS project locally:
+
+```bash
+yarn dev
+```
+```bash
+npm start
+```
+Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+
+### Build and Production
+
+To build the application:
+
+```bash
+yarn build
+```
+```bash
+npm run build
+```
+
+After building, to run it in production mode:
+
+```bash
+yarn start
+```
+
+```bash
+npm start
+```
+### Linting
+
+To lint your code:
+
+```bash
+yarn lint
+```
+```bash
+npm run lint
+```
+
+## Pulling .env Variables from Vercel
+
+**1. Install Vercel CLI**
+If you haven't already, install the [Vercel CLI](https://vercel.com/docs/cli#using-in-a-ci/cd-environment):
+
+```bash
+npm i -g vercel
+```
+```bash
+yarn global add vercel
+```
+
+**2. Log into Your Vercel Account**
+From the command line, log into your Vercel account:
+
+```bash
+vercel login
+```
+Vercel CLI supports the following login methods:
+```bash
+GitHub OAuth
+GitLab OAuth
+Bitbucket OAuth
+Email confirmation
+SAML Single Sign-On through your Team's identity provider
+```
+Vercel CLI requires you to log in and authenticate before accessing resources or performing administrative tasks. In a terminal environment, you can use vercel login, which requires manual input.
+This will prompt you for your email address and send you a confirmation email. Once you click on the link in the email, the CLI login process will be completed.
+
+**3. Navigate to Your Project**
+Navigate to your project's root directory:
+
+```bash
+cd path-to/your-project-directory
+```
+
+**4. Pull the .env Variables Locally**
+Use the following command to pull the environment variables from Vercel to a local .env file:
+
+```bash
+vercel env pull 
+```
+
+This will write the environment variables from Vercel to a .env file in your local environment.
+
+For futher visit [Vercel](https://vercel.com/docs/cli/env).
+
+
+
+## Using ShadCn
+ShadCn provides a streamlined way to add components to your projects. Here's a comprehensive guide on how to use it.
+
+### Command Line Interface (CLI)
+The CLI is your main tool for interacting with ShadCn.
+
+Initializing a New Project
```

Comment:
its already initialized right? can just explain how to add new components

Reply:
Initialization part removed.